core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/common/Status.java

```diff
@@ -50,7 +50,11 @@
     EXTERNAL_DATABASE_START_FAILED(StatusKind.FINAL),
     EXTERNAL_DATABASE_STOP_IN_PROGRESS(StatusKind.PROGRESS),
     EXTERNAL_DATABASE_STOP_FINISHED(StatusKind.PROGRESS),
-    EXTERNAL_DATABASE_STOP_FAILED(StatusKind.FINAL);
+    EXTERNAL_DATABASE_STOP_FAILED(StatusKind.FINAL),
+    LOAD_BALANCER_UPDATE_IN_PROGRESS(StatusKind.PROGRESS),
```

Comment:
this is probably requires UI changes

Reply:
Opened https://jira.cloudera.com/browse/CB-11308 for Santiago to work on the UI changes.